packages/flutter/lib/src/widgets/implicit_animations.dart

```diff
@@ -331,7 +335,19 @@ abstract class ImplicitlyAnimatedWidgetState<T extends ImplicitlyAnimatedWidget>
   /// 2. Take the value returned from the callback, and store it. This is the
   /// value to use as the current value the next time that the forEachTween()
   /// method is called.
+  ///
+  /// Subclasses that contain properties based on tweens created by
+  /// forEachTween() should override onTweensUpdated() to update those
+  /// properties. Dependent properties should not be updated within
+  /// forEachTween().
   void forEachTween(TweenVisitor<dynamic> visitor);
+
+  /// Optional hook for subclasses that runs after all tweens have been updated
+  /// via forEachTween().
+  ///
+  /// Any properties that depend upon tweens created by forEachTween() should be
+  /// updated within onTweensUpdated(), not within forEachTween().
+  void onTweensUpdated() {}
```

Comment:
See https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#naming-rules-for-typedefs-and-function-variables
This should be called something verb-like. "setupAnimations" or "didUpdateTweens" or some such.